[TektonTarget]Target makes controller panic

TektonTarget seems to make controller panic.

Tekton object
```yaml
apiVersion: targets.triggermesh.io/v1alpha1
kind: TektonTarget
metadata:
  creationTimestamp: "2022-06-24T07:13:23Z"
  generation: 1
  labels:
    flow.triggermesh.io/bridge-watched: "true"
    flow.triggermesh.io/created-by: ast-twilio-tekton-tzc
    flow.triggermesh.io/used-by.ast-twilio-tekton-tzc: dominant
  name: a-tekton-swim
  namespace: personal-org-29-personal-ns-25
  resourceVersion: "1472322941"
  uid: 7b6fd9f4-28f4-470a-b309-1af0f9e7e165
spec:
  reapPolicy:
    success: 363m
```

Controller logs
```
I0627 13:01:57.681342       1 leaderelection.go:258] successfully acquired lease triggermesh/triggermesh-controller.github.com.triggermesh.triggermesh.pkg.targets.reconciler.tektontarget.reconciler.00-of-01
{"level":"info","ts":"2022-06-27T13:01:57.681Z","logger":"triggermesh-controller","caller":"leaderelection/context.go:158","msg":"\"triggermesh-controller-d45b74b9c-llclq_d2ce3acf-3bf4-4148-aa9f-be4d7347a3c7\" has started leading \"triggermesh-controller.github.com.triggermesh.triggermesh.pkg.targets.reconciler.tektontarget.reconciler.00-of-01\"","commit":"130294b"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x22d948b]

goroutine 5455 [running]:
github.com/triggermesh/triggermesh/pkg/targets/reconciler/tektontarget.makeAppEnv(...)
        github.com/triggermesh/triggermesh/pkg/targets/reconciler/tektontarget/adapter.go:71
github.com/triggermesh/triggermesh/pkg/targets/reconciler/tektontarget.(*Reconciler).BuildAdapter(0xc001465ac0, {0x307f118?, 0xc002a2b040}, 0x30?)
        github.com/triggermesh/triggermesh/pkg/targets/reconciler/tektontarget/adapter.go:52 +0x1eb
github.com/triggermesh/triggermesh/pkg/reconciler.(*GenericServiceReconciler[...]).ReconcileAdapter(0xc001465ac0, {0x3047c68, 0xc003d4bc50}, {0x301a540, 0xc001465ac0})
        github.com/triggermesh/triggermesh/pkg/reconciler/reconcile.go:176 +0xec
github.com/triggermesh/triggermesh/pkg/targets/reconciler/tektontarget.(*Reconciler).ReconcileKind(0xc001465ac0, {0x3047c68?, 0xc003d4bb60?}, 0xc002a2b168?)
        github.com/triggermesh/triggermesh/pkg/targets/reconciler/tektontarget/reconciler.go:47 +0x6a
github.com/triggermesh/triggermesh/pkg/client/generated/injection/reconciler/targets/v1alpha1/tektontarget.(*reconcilerImpl).Reconcile(0xc0004a8a00, {0x3047c68, 0xc003d4bb30}, {0xc005757f50, 0x2c})
        github.com/triggermesh/triggermesh/pkg/client/generated/injection/reconciler/targets/v1alpha1/tektontarget/reconciler.go:239 +0x5b7
knative.dev/pkg/controller.(*Impl).processNextWorkItem(0xc0014ceea0)
        knative.dev/pkg@v0.0.0-20220525153005-18f69958870f/controller/controller.go:542 +0x58d
knative.dev/pkg/controller.(*Impl).RunContext.func3()
        knative.dev/pkg@v0.0.0-20220525153005-18f69958870f/controller/controller.go:491 +0x68
created by knative.dev/pkg/controller.(*Impl).RunContext
        knative.dev/pkg@v0.0.0-20220525153005-18f69958870f/controller/controller.go:489 +0x312

```

--- a/pkg/targets/reconciler/tektontarget/adapter.go
+++ b/pkg/targets/reconciler/tektontarget/adapter.go
@@ -65,7 +65,7 @@ func makeAppEnv(o *v1alpha1.TektonTarget) []corev1.EnvVar {
 			})
 		}
 
-		if o.Spec.ReapPolicy.ReapSuccessAge != nil {
+		if o.Spec.ReapPolicy.ReapFailAge != nil {
 			envVar = append(envVar, corev1.EnvVar{
 				Name:  "TEKTON_REAP_FAIL_AGE",
 				Value: *o.Spec.ReapPolicy.ReapFailAge,

